tests/kubernetes/kubernetes_cluster_connector_test.py

```diff
@@ -132,13 +131,12 @@ def test_total_cpus(mock_cluster_connector):
     assert mock_cluster_connector.get_resource_total('cpus') == 12
 
 
-def test_get_pending_pods(mock_cluster_connector):
-    assert len(mock_cluster_connector._get_pending_pods()) == 1
-
-
-def test_get_unschedulable_pods(mock_cluster_connector):
-    assert len(mock_cluster_connector.get_unschedulable_pods()) == 1
-
-
-def test_pending_cpus(mock_cluster_connector):
-    assert mock_cluster_connector.get_resource_pending('cpus') == 1.5
+def test_delete_node(mock_cluster_connector):
```

Comment:
nit: you might want to do the similar tests for evict_pods_on_node and set_node_unschedulable